SPECS/ocaml-bisect-summary.spec

```diff
@@ -0,0 +1,40 @@
+
+Name:           ocaml-bisect-summary
+Version:        0.4
+Release:        1%{?dist}
+Summary:        Report code coverage from bisect_ppx runtime files
+License:        LGPL
+URL:            https://github.com/lindig/bisect-summary
+Source0:        https://github.com/lindig/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  oasis-devel
```

Comment:
I agree that `bisect-summary` is a better name and will change it. I believe that I started with `oasis` but needed `oasis-devel` for the setup step.